config/web.js

```diff
@@ -61,7 +61,7 @@ module.exports = {
       }
       // respond with json
       else if (req.wantsJSON) {
-        res.send({error: error})
+        res.send(error)
```

Comment:
Hum in this case we need more smart test because sometime `error` is undefined or it's just a string so we can't send it directly...
What about this : 
If it's an object just do like you did and if it's not wrap it with {error:error} 
How does it sound ?